test-server/index.js

```diff
@@ -154,7 +154,8 @@ export class TestServer {
         // istanbul ignore next
         throw new Error(
           `Client was not subscribed to ${action.channel} ` +
-            'but it try to unsubscribe from it'
+            (action.filter ? `with filter ${action.filter} ` : '') +
```

Comment:
Filter is an object.

```suggestion
            (action.filter ? `with filter ${JSON.stringify(action.filter)} ` : '') +
```